### PR TITLE
update perception functionality benchmark

### PR DESCRIPTION
**Pull request by mhwasil**

### CODE REVIEW

File: rulebook/titlepageRulebook.tex [hunk 1]
```diff
@@ -60,7 +60,7 @@
 	\revisionNumberRuleBook\\
 	\today\\
 	\mbox{~}\\
-	Deebul Nair, Gerhard Kraetzschmar, Sven Schneider\\
+	Mohammad Wasil, Deebul Nair, Gerhard Kraetzschmar, Sven Schneider\\
```
Comment (deebuls) [inline rulebook/titlepageRulebook.tex]: Add Paul G. Ploeger remove Kraetzschmar

**Review by deebuls — CHANGES_REQUESTED** [review]
Add Ploeger name and remove kraeztschmar

File: rulebook/titlepageRulebook.tex (hunk 1 above)
Comment (mhwasil) [reply rulebook/titlepageRulebook.tex]: should we move him to contributors?

Comment (mhwasil) [reply rulebook/titlepageRulebook.tex]: he is already in contributors anyway

**Review by deebuls — APPROVED** [review]

**Comment by deebuls** [comment]
@mhwasil good work and thank you !

**Pull request merged**